README.md

```diff
@@ -601,16 +626,50 @@ WebHttpRequest.builder()
         .build()
 ```
 
-### Send http request through WebHttpClient
+### Send the HTTP request through WebHttpClient
+``` java
+client.sendAsync(request);
+```
+
+Calling the sendAsync method returns a WebHttpResponse object. 
+
+### Retrieving the HTTP Response through WebHttpResponse 
+
+The methods provided to retrieve information from WebHttpReponse are: 
+- `getStatus()`
+- `getMessage()` 
+- `getBody()`
+- `getHeaders()`
+
+An example: 
+
+``` java
+WebHttpResponse response = client.sendAsync(request); 
+String responseBody = response.getBody(); 
```

Comment:
Is this one correct? I thought send async will return `CompletableFuture<WebHttpResponse>`?

Reply:
Something like:

```java
        client.sendAsync(request)
                .thenApply(response -> {
                    final String responseBody = response.getBody();
                    // do processing;
                })
                .exceptionally(e -> {
                   // error handling 
                 });
```

Reply:
Thank you Yudhi, I have updated the README :)